drivers/flash/flash_stm32g4x.c

```diff
@@ -113,7 +132,7 @@ static int erase_page(struct device *dev, unsigned int page)
 	/* Wait for the BSY bit */
 	rc = flash_stm32_wait_flash_idle(dev);
 
-	regs->CR &= ~FLASH_CR_PER;
```

Comment:
Please put this under #ifdef FLASH_OPTR_DBANK

Reply:
It does not change anything. The datasheet tells to keep FLASH_CR_BKER at reset value (false).

Reply:
Oh, I see. Some MCUs don't have that bit!

Reply:
Done